site/src/components/WorkspaceStatusBadge/WorkspaceStatusBadge.tsx

```diff
@@ -34,70 +21,72 @@ export const getStatus = (
   icon: React.ReactNode
 } => {
   const status = getWorkspaceStatus(build)
+  const { t } = i18next
+
   switch (status) {
     case undefined:
       return {
-        text: StatusLanguage.loading,
+        text: t("common:workspace_status.loading", { ns: "common" }),
```

Comment:
Here's another way to namespace. There's actually a third way to namespace: I could write
`t("common:workspace_status.loading")`, but some people find this more difficult to read. I don't have a preference myself.